eman2/protocols/protocol_tomo_extraction.py

```diff
@@ -24,11 +24,17 @@
 # *
 # **************************************************************************
 
+import os
+
 from pyworkflow import utils as pwutils
 import pyworkflow.em as pwem
 import pyworkflow.protocol.params as params
+
 from tomo.protocols import ProtTomoBase
 from tomo.objects import SetOfSubTomograms, SubTomogram
+
+from pyworkflow.utils.path import moveFile, cleanPath, makePath
```

Comment:
Please have a look at this comment and follow this rule: https://github.com/scipion-em/scipion-em-tomo/pull/1#discussion_r271045909